documentation/rfc/line_sections.md

```diff
@@ -0,0 +1,165 @@
+
+On ptref we can display the line section disruption on a ptref call if the last object of the ptref call is directly linked to an **active** disruption
+
+examples:
+
+this is the navitia call and YES means we display the disruption, NO means we do not display it
+
+`/routes/route_1_of_the_line_1` --> YES (? not sure about this one if the line section is represented as linked to the line)
+
+`/stop_areas/A`  --> NO
+`/stop_areas/B`  --> NO
+`/stop_areas/C`  --> YES
+`/stop_areas/D`  --> YES
+`/stop_areas/E`  --> YES
+`/stop_areas/F`  --> NO
+
+`/routes/route_2_of_the_line_1`  --> NO
+
+`/lines/line_1`  -->  YES (? not sure about this one if we represent the line section as linked to 2 stops and a list of routes)
+`/lines/line_2`  -->  NO
+
+If we decide to display the line section disruptions based on the last object of the query it will lead to some difficult to understand behaviour:
+
+`/stop_areas/A/routes/route_1_of_the_line_1`  --> YES
+`/stop_areas/C/routes/route_2_of_the_line_1`  --> NO
+
+`/networks/network_of_the_route_1 `  --> NO
+
+`/stop_points/C_1`  --> NO
+`/vehicle_journeys/vj1_of_route_1_of_line_1`  --> NO
+
+
+### ptref with /disruptions
+
+This is for the calls api.navitia.io/v1/coverage/bob/<some pt ref filters>/disruptions
+
+The /disruption api is meant to be a direct representation of the object model below so I think it's ok the keep the same mechanism as the common cases (but not restricted ot the active disruptions by default)
+
+### ptref with /traffic_reports
+
+This is for the calls api.navitia.io/v1/coverage/bob/<some pt ref filters>/traffic_reports
+
+This /traffic_reports api is meant to be a clever representation of the disruptions active on the object and thus we can do more things on this
+
+`/stop_areas/B/traffic_reports`  --> NO
+`/stop_areas/C/traffic_reports`  --> YES
+
+`/lines/line_1/traffic_reports ` -->  YES
+`/lines/line_2/traffic_reports`  -->  NO
+
+`/routes/route_1_of_the_line_1/traffic_reports`  --> YES
+`/routes/route_2_of_the_line_1/traffic_reports`  --> NO
+
+`/stop_areas/A/routes/route_1_of_the_line_1/traffic_reports`  --> YES (I think NO would be better it but this will be quite difficult for not that much gain)
+`/stop_areas/C/routes/route_2_of_the_line_1/traffic_reports`  --> NO
+
+`/networks/network_of_the_route_1/traffic_reports `  --> YES
+`/stop_points/C_1/traffic_reports`  --> YES since it's stop_area is part of an impacted line section
+
+`/vehicle_journeys/vj1_of_route_1_of_line_1/traffic_reports`  --> NO (It would be better to display it, but we do not display other disruptions there, would it be worth it to change this ?)
+
+
+## schedules
+
+I don't think we need to do some stuff here, the same rules as the common cases can be used.
+
+## journeys
+
+If a journey use a VJ of an impacted route on at least one stop point impacted by the line section we display it.
+
+
+# modelisation
+
+TODO, we need to first agree on the other questions before doing this
```

Comment:
This summary of differences clarifies the doc and it would be nice to add it somewhere, to understand better differences between disruptions vs traffic_reports.